server/plugin/plugin.go

```diff
@@ -158,6 +168,27 @@ func (p *MatterpollPlugin) setProfileImage() error {
 	return nil
 }
 
+// checkHTTPConnection checks if a http request to
+func (p *MatterpollPlugin) checkHTTPConnection() error {
+	siteURL := *p.ServerConfig.ServiceSettings.SiteURL
+	url := siteURL + "/plugins/" + manifest.Id + "/"
+
+	for i := 0; i < 3; i++ {
+		resp, err := http.Get(url)
+		if err != nil {
+			p.API.LogDebug("failed to make http request to root plugin URL", "URL", url, "error", err.Error())
+			time.Sleep(1 * time.Second)
```

Comment:
Shouldn't it be retried when http.StatusNotFound is returned?
Because if sending the request before running plugin server, I supposed that server doesn't return  errors, and return `404 NotFound`.

Reply:
I agree that we should retry when `http.StatusNotFound` is returned. But shouldn't we also retry when an error is returned?

Reply:
Since retry is for waiting for the server to start, I think the we don't need to retry when catching err. 1/5